doc/dynamo0p3.rst

```diff
@@ -1191,22 +1253,36 @@ plus_fields
 
 Sums two fields: ::
   
-  field3(:) = field1(:) + field2(:)
+   field3(:) = field1(:) + field2(:)
 
 where:
 
 * type(field_type), intent(in) :: *field1*
 * type(field_type), intent(in) :: *field2*
 * type(field_type), intent(out) :: *field3*
 
+raise_field
++++++++++++
+
+**raise_field** (*field1*, *scalar*)
+
+Raises a field to a scalar value and returns the field: ::
+
+   field1(:) = field1(:) **  scalar
+
+where:
+
+* type(field_type), intent(inout) :: *field1*
+* real(r_def),      intent(in) :: *scalar*
+
 scale_field
 +++++++++++
 
 **scale_field** (*scalar*, *field1*)
 
 Multiplies a field by a scalar value and returns the field: ::
 
-  field1(:) = scalar * field1(:)
+   field1(:) = scalar * field1(:)
```

Comment:
Deleted extra spaces around "mult" (*), and also around "divide" (/) and raise field (**) for sake of consistency. Left spaces around "-" and "+".